hathor/indexes/manager.py

```diff
@@ -147,3 +150,14 @@ def del_tx(self, tx: BaseTransaction, *, remove_all: bool = False, relax_assert:
 
         if self.tokens:
             self.tokens.del_tx(tx)
+
+
+class RocksDBIndexesManager(MemoryIndexesManager):
```

Comment:
I think this is a consequence of why I think the abstraction on `IndexesManager` isn't good. But anyway I'm not opposed to merging this as is. I'll make a proposal to refactor this tomorrow if this is merged.

Reply:
The main thing that you don't like is that we should be extending `IndexesManager` here and not `MemoryIndexesManager`?

Reply:
Not really. I think `IndexesManager` isn't really being usefully abstracted here, nor it will be after all on-disk indexes are implemented. This is how the `TransactionStorage` is right now, and we ended up growing a lot of complexity because of that. IMO we shouldn't do the same here.

A better approach IMO is to leave `IndexesManager` as a concrete class that has abstract properties on the signature but the concrete properties are instantiated by a builder (looks like a factory but is meant to construct a single instances, not many/repeated instances). This would solve the awkward instantiation inside the `enable_*` methods (that currently is the only abstraction we're actually using).

Reply:
I think I agree with you. I guess we should leave this refactor for another PR. What do you think?